OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs

```diff
@@ -52,7 +52,21 @@ public void Run(ModData modData, string[] args)
 				var maps = new List<Map>();
 				if (args.Length < 2)
 				{
-					maps.Add(null);
+					Console.WriteLine("Testing mod: {0}".F(Game.ModData.Manifest.Mod.Title));
+					CheckGameRules(Game.ModData.DefaultRules);
```

Comment:
Please add a comment like "Run all rule checks on the default mod rules." (although somewhat obvious), and perhaps move it after the `foreach`?